asv_bench/benchmarks/io/csv.py

```diff
@@ -69,6 +69,7 @@ def setup(self, infer_datetime_format, format):
         self.data = StringIO('\n'.join(rng.strftime(dt_format).tolist()))
 
     def time_read_csv(self, infer_datetime_format, format):
+        self.data.seek(0)
```

Comment:
hmm can we instead make ``data`` a property which does this, otherwise this is likely to be forgotten the next time a benchmark is written. can you show a before /after on these benchmarks?